### ISSUE TITLE
Can't Login Using the Enter Key

### ISSUE THREAD

**Issue by danlerche** [Libki/libki-client#84]
It seems the only way for a patron to log is by using mouse and clicking on the the Log In button. Pressing hitting while tabbing over to the Log in button doesn't work for me. This has accessibility ramifications, as patrons should be able to use our devices by just using the keyboard.

**Comment by kylemhall** [comment]
What platform are you on? What client version are you running? I cannot recreate this issue on Windows 10 with the latest client build.

**Comment by danlerche** [comment]
I believe I was using the latest one on the list (Client unstable). How can I check the version once installed? Installed on both a linux and windows client.

**Comment by danlerche** [comment]
OK just tried latest and I'm getting the same results: https://github.com/Libki/libki-client/releases/tag/r20.11

**Comment by kylemhall** [comment]
@danlerche same results as you had previously, or same as mine?

If you mean still borked for you, please try the latest unstable ( https://github.com/Libki/libki-client/releases ). The client version number should be in the bottom right of the login screen.

**Comment by kylemhall** [comment]
Also, what OS are you running on?

**Comment by danlerche** [comment]
@kylemhall Running windows 10 (had same issue on Linux mint). Still borked for me unfortunately using multiple versions. Even tried switching out the keyboard. I might try installing to a different location in case it's an older file getting in the way.

**Comment by danlerche** [comment]
OK I found some interesting behavior: If you type in the username (library card), then hit enter while still in that field, then tab to the log in screen, it works. But if you type in the username, and tab into the log in button, it doesn't register the enter key.

**Comment by kylemhall** [comment]
Ah, for windows and web UIs even, I believe the standard trigger is not enter/return, but the space bar!
The good news is, you don't *need* to use that button! Just type a username, hit enter/return, type a password, and hit enter/return from the password field. Hitting enter/return from the password field will act just like clicking the "Log In" button.

**Comment by danlerche** [comment]
I actually didn't know space bar was the trigger! I'll look into that more. Also, if we're not requiring a password (pin), and we decide to hide that field, will it still work?

**Comment by danlerche** [comment]
> 

Never mind. I see it works!

**Comment by kylemhall** [comment]
Yes, there is a client ini config option to hide the password field.
